mlfinlab/tests/test_feature_importance.py

```diff
@@ -214,6 +220,48 @@ def test_feature_importance(self):
         self.assertAlmostEqual(sfi_feat_imp_f1.loc['label_prob_0.2', 'mean'], 0.74, delta=1)
         self.assertAlmostEqual(sfi_feat_imp_f1.loc['label_prob_0.5_sma_2', 'mean'], 0.224, delta=1)
 
+        #CFI(log_loss) assertions
+        self.assertAlmostEqual(cfi_feat_imp_linear.loc['label_prob_0.1', 'mean'], 0.2, delta=3)
+        self.assertAlmostEqual(cfi_feat_imp_linear.loc['label_prob_0.2', 'mean'], 0.3, delta=3)
+
+    def test_value_error_raise(self):
+        """
+        Test get_feature_clusters , codependence_matrix and distance_matrix for invalid arguments
+        """
+        #Unkown dependence_metric
+        with self.assertRaises(ValueError):
+            get_feature_clusters(self.X_train, dependence_metric='information',
+                                 distance_metric='angular', linkage_method='single',
+                                 n_clusters=2)
+        #Unkown distance_metric
+        with self.assertRaises(ValueError):
+            get_feature_clusters(self.X_train, dependence_metric='linear',
+                                 distance_metric='serial', linkage_method='single',
+                                 n_clusters=2)
+        #Number of clusters larger than number of features
+        with self.assertRaises(ValueError):
+            get_feature_clusters(self.X_train, dependence_metric='linear',
+                                 distance_metric='angular', linkage_method='single',
+                                 n_clusters=int(len(self.X_train)))
+
+    def test_get_feature_clusters(self):
```

Comment:
Let's put feature clusters tests to a separate file

Reply:
That can be done.